UQ-eLIPSE/crucible-components
# story/VETHUB-28

### I should be able to revisit questions I skipped

To test:

- yarn dev
- Navigate to http://localhost:5173/

Verify:

- Navigating the questions is enabled either by submitting or skipping.
- If you skip all questions, that will result in a circular navigation (you will go back to question 0).
- If you answer and submit all questions, that will take you to a screen without a question.
- If you answer some questions and skip some other, those questions you skipped will be displayed again. 

Run tests:

- yarn run test
